tests/test_config_consistency.py

```diff
@@ -281,6 +281,11 @@ class Test_Config_UnifiedServiceTagging_CustomService:
     def setup_specified_service_name(self):
         self.r = weblog.get("/")
 
+    @irrelevant(
```

Comment:
The provided reason makes me think it's a bug, no ? 😉

Reply:
Gin integration requires that you provide it a service name, which in practice will conflict with DD_SERVICE...Yes, maybe we want to change gin integration to not require service name, but it's not breaking anything per se... So I would not call it a bug... what do you think? 🤔

Reply:
The boundary here is blurry. From what you say, I understand you plan to leave it that way, so irrelevant is fine.